svc-api/handle/handlers_test.go

```diff
@@ -517,6 +520,16 @@ func TestChassisMethodNotAllowed(t *testing.T) {
 	e.PATCH("/redfish/v1/Chassis/" + chassisID + "/PCIeDevices/" + rID).Expect().Status(http.StatusMethodNotAllowed)
 	e.DELETE("/redfish/v1/Chassis/" + chassisID + "/PCIeDevices/" + rID).Expect().Status(http.StatusMethodNotAllowed)
 
+	e.POST("/redfish/v1/Chassis/" + chassisID + "/PCIeDevices/" + rID + "/PCIeFunctions").Expect().Status(http.StatusMethodNotAllowed)

```

Comment:
Normally we check methods only for the supported ones and all unsupported will be used with "Any"